.github/workflows/build.yml

```diff
@@ -125,3 +130,42 @@ jobs:
           tag_name: ${{ github.event.create.ref }}
           draft: false
           prerelease: ${{ steps.gitversion.outputs.preReleaseTag != '' }}
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v1
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ github.token }}
+
+      - name: Extract Metadata for Docker
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ github.repository }},ghcr.io/${{ github.repository }}
+          tags: type=ref,event=tag
+
+      - name: Build & Push Image
+        uses: docker/build-push-action@v2
+        with:
+          push: true
+          context: ./docker
+          build-args: |
+            DOTNET_VERSION=${{ env.DOTNET_VERSION }}
+            REPOSITORY=${{ github.repository }}
+            VERSION=${{ github.ref_name }}
+          platforms: linux/arm/v7,linux/arm64,linux/amd64
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
```

Comment:
I'd like to see a separate workflow for this with a dependency. This has the benefit of keeping this YAML file small.

For example, look at the [notify.yml](https://github.com/rcdailey/trash-updater/blob/master/.github/workflows/notify.yml) workflow. That one is triggered any time a release is published.

Reply:
Will do.